app/controllers/api/v3/npq_applications_controller.rb

```diff
@@ -19,6 +19,17 @@ def index
       def json_serializer_class
         Api::V3::NPQApplicationSerializer
       end
+
+      def query_scope
+        scope = super
```

Comment:
We introduced new patterns in v3 where we moved queries out of controllers into query services, maybe we can do something similar here and move this into it's own query service with specs? please find an example of another query service [here](https://github.com/DFE-Digital/early-careers-framework/blob/main/app/services/api/v3/npq_participants_query.rb).